deploy/common/k8sbased.sh

```diff
@@ -341,10 +341,6 @@ function launch_central {
         ${ORCH_CMD} -n stackrox set env deploy/central MODULE_LOGLEVELS="${MODULE_LOGLEVELS}"
       fi
 
-      if [[ "$ROX_MANAGED_CENTRAL" == "true" ]]; then
```

Comment:
Do you just want to error out here, and indicate that deploying with `ROX_MANAGED_CENTRAL=true` is only supported when choosing the Helm route (`OUTPUT_FORMAT=helm`)?